db/re/item_db_usable.yml

```diff
@@ -35035,6 +35122,351 @@ Body:
       NoAuction: true
     Script: |
       getgroupitem(IG_July_Lucky_Scroll);
+  - Id: 14841
+    AegisName: DF_Songpyun
+    Name: "[Scroll] Korean Rice Cake"
+    Type: Healing
+    Buy: 2
+    Weight: 10
+    NoUse:
+      Override: 100
+      Sitting: true
+    Trade:
+      Override: 100
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      percentheal 10,0;
+  - Id: 14843
+    AegisName: DF_Insurance
+    Name: "[Scroll] Life Insurance"
+    Type: Usable
+    Buy: 2
+    Weight: 10
+    NoUse:
+      Override: 100
+      Sitting: true
+    Trade:
+      Override: 100
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      sc_start SC_LIFEINSURANCE,1800000,0;
+  - Id: 14844
+    AegisName: DF_Kafra_Card
+    Name: "[Scroll] Kafra Card"
+    Type: Usable
+    Buy: 2
+    Weight: 10
+    NoUse:
+      Override: 100
+      Sitting: true
+    Trade:
+      Override: 100
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      callfunc "F_CashStore";
+  - Id: 14845
+    AegisName: DF_Blessing_10_Scroll
+    Name: "[Scroll] Blessing Lv.10 Scroll"
+    Type: Usable
+    Buy: 2
+    Weight: 10
+    NoUse:
+      Override: 100
+      Sitting: true
+    Trade:
+      Override: 100
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      skilleffect "AL_BLESSING",0;
+      sc_start SC_BLESSING,240000,10;
```

Comment:
Same, for me the item should look like this one
```
  - Id: 12522
    AegisName: E_Blessing_10_Scroll
    Name: Blessing Scroll Lv 10
    Type: Delayconsume
    Buy: 2
    Weight: 10
    Trade:
      Override: 100
      NoDrop: true
      NoTrade: true
      NoSell: true
      NoCart: true
      NoGuildStorage: true
      NoMail: true
      NoAuction: true
    Script: |
      itemskill "AL_BLESSING",10;
```

Reply:
Thanks!